crates/cargo-wdk/src/actions/new/mod.rs

```diff
@@ -158,7 +150,7 @@ impl<'a> NewAction<'a> {
         let template_file = TEMPLATES_DIR.get_file(&template_path).ok_or_else(|| {
             NewActionError::TemplateNotFound(template_path.to_string_lossy().into_owned())
         })?;
-        let lib_rs_path = self.cwd.join("src/lib.rs");
+        let lib_rs_path = self.path.join("src/lib.rs");
```

Comment:
`.join("src/lib.rs");` I hope join can already handle this irrespective forward slash or backslash in the string. But it is better to write it like this `.join("src").join("lib.rs");`